browse-everything.gemspec

```diff
@@ -28,7 +28,7 @@ Gem::Specification.new do |spec|
   spec.add_dependency 'rails', '>= 4.2', '< 6.0'
   spec.add_dependency 'ruby-box'
   spec.add_dependency 'signet', '~> 0.8'
-  spec.add_dependency 'sprockets', '~> 3.7'
+  spec.add_dependency 'sprockets', '>= 3.7', "< 5"
```

Comment:
actually i'm not sure if we should remove this dependency entirely. It was not there at all until 5685a4ad49. Thoughts?